tools/bulk_compression.py

```diff
@@ -0,0 +1,405 @@
+
+def returnFilteredPwdContents(file_extension):                      # Get files in pwd with extension
+    extension_pattern = r".*\." + file_extension.lower()
+    extension_reg = re.compile(extension_pattern)
+    return [fname for fname in os.listdir('.') if extension_reg.match(fname)]
+
+# -------------------------------------------------------------------------------------------------
+
+def checkSuccess(decompressing, fname, extension):                  # Ensure file created properly
+
+    if (not exists(f"./{fname}.{extension}")):
+
+        print("╠===============================================================================╣")
+        if (decompressing):
+            print(f"║ {extension.upper()} decompress failed.{(61 - len(extension)) * ' '}║")
+            print(f"║ Extracted file {fname}.{extension} missing or empty.{(26 - (len(extension) * 2) - len(fname)) * ' '}║")
+        else:
+            print(f"║ {extension.upper()} compress failed.{(61 - len(extension)) * ' '}║")
+            print(f"║ New file {fname}.{extension} missing or empty.{(50 - len(extension) - len(fname)) * ' '}║")
+        print("╚===============================================================================╝")
+        sys.exit(1)
+
+    print(f"║ {fname}.{extension} successfully created.{(55 - len(fname) - len(extension)) * ' '}║")
+
+# -------------------------------------------------------------------------------------------------
+
+def checkProgramMissing(program):
+
+    if (sys.platform.startswith('win32') and exists(f"./{program}.exe")):
+        return
+
+    else:
+        from shutil import which
+        if (which(program) is not None):
+            return
+
+    print("║                                                                                ")
+    print("╠===============================================================================╗")
+    print(f"║                   {program} failed, {program} file is missing.{(34 - (len(program) * 2)) * ' '}║")
+    print("╚===============================================================================╝")
+    sys.exit(1)
+
+# -------------------------------------------------------------------------------------------------
+
+def printBinCueMismatch():
+    print("║                                                                                ")
+    print("╠===============================================================================╗")
+    print(f"║                   All BIN files must have a matching CUE.                     ║")
+    print("╚===============================================================================╝")
+    sys.exit(1)
+
+# -------------------------------------------------------------------------------------------------
+
+def checkBinCueMismatch(bin_files, cue_files):                      # Ensure all bins and cues match
+    if (len(bin_files) == len(cue_files)):
+        for i in range(len(bin_files)):
```

Comment:
Can't you instead do `for val in bin_files`?
Codacy recommends enumerate() but you don't actually use the index outside of indexing into `bin_files`